feature-libs/my-account/organization/assets/translations/en/user-group.i18n.ts

```diff
@@ -1,62 +1,90 @@
 export const userGroup = {
-  header: 'User groups',
+  header: 'All user groups ({{count}})',
   disabled: '(disabled)',
   uid: 'Code',
   name: 'Name',
-  orgUnit: 'Parent Unit',
-
-  messages: {
-    enabled:
-      'When you disable the user group, the related data will be disabled as well. ',
-    disabled: 'You cannot edit a disabled User Group.',
-    deactivate: 'Are you sure you want to disable this user group?',
-    deactivateHeader: 'Disable User Group',
+  unit: 'Unit',
+  orgUnit: 'Unit',
+  actions: '',
+
+  byName: 'Sort by name',
+  byUnitName: 'Sort by unit',
+  byGroupID: 'Sort by group',
+
+  details: {
+    title: 'User group Details',
+    subtitle: 'User group: {{ item.name }}',
+  },
+
+  edit: {
+    title: 'Edit User group',
+    subtitle: 'User group: {{ item.name }}',
+  },
+
+  create: {
+    title: 'Create User group',
+    subtitle: '',
   },
 
   user: {
     link: 'Users',
-    header: 'Users in {{code}}',
-    assign: 'Assign users',
-    assignHeader: 'Assign users in {{code}}',
-    unassignAll: 'Unassign All',
-    back: 'Back',
-    instructions: {
-      check: 'To assign a user to this user group, select its check box.',
-      uncheck: 'To unassign a user, clear its check box.',
-      changes: 'Changes are saved automatically.',
-    },
+    //   header: 'Users in {{code}}',
```

Comment:
What will we do with those comments? Will these translations be useful in the future?

Reply:
I've cleaned it up